app/containers/MessageBox/index.js

```diff
@@ -352,60 +353,33 @@ class MessageBox extends Component {
 	}
 
 	// eslint-disable-next-line react/sort-comp
-	debouncedOnChangeText = debounce(async(text) => {
-		const { sharing } = this.props;
-		const db = database.active;
+	debouncedOnChangeText = debounce((text) => {
 		const isTextEmpty = text.length === 0;
-		// this.setShowSend(!isTextEmpty);
-		this.handleTyping(!isTextEmpty);
-
-		if (!sharing) {
-			// matches if their is text that stats with '/' and group the command and params so we can use it "/command params"
-			const slashCommand = text.match(/^\/([a-z0-9._-]+) (.+)/im);
-			if (slashCommand) {
```

Comment:
Some slash commands run while typing, some run when the message is sent.
While typing, we need to evaluate the kind of command and act as expected.
If you try `/giphy` on https://open.rocket.chat, you'll see that you can play with gif previews while sending a message.
Open the app and type: `/giphy dog`